docs/deploy.md

```diff
@@ -17,17 +17,19 @@
 #
 -->
 
-# Deploy manually
+# Deploy Manually
```

Comment:
Only the first word should begin with a uppercase letter normally.

Reply:
done